CIP-XXXX/cardano-babbage.json

```diff
@@ -0,0 +1,1826 @@
+      "additionalProperties": false
+    },
+    "PlutusScript": {
+      "title": "PlutusScript",
+      "type": "object",
+      "properties": {
+        "language": {
+          "$ref": "cardano-babbage.json#/definitions/Language"
+        },
+        "bytes": {
+          "type": "string",
+          "format": "hex",
+          "pattern": "^([0-9a-f][0-9a-f])+$"
+        }
+      },
+      "required": ["language", "bytes"],
+      "additionalProperties": false
+    },
+    "NativeScript": {
+      "title": "NativeScript",
+      "discriminator": {
+        "propertyName": "tag"
+      },
+      "oneOf": [
+        {
+          "title": "ScriptPubkey",
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "pubkey"
+              ]
+            },
+            "pubkey": {
+              "$ref": "cardano-babbage.json#/definitions/Ed25519KeyHash"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["tag", "pubkey"]
+        },
+        {
+          "title": "ScriptAll",
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "all"
+              ]
+            },
+            "scripts": {
+              "type": "array",
+              "items": {
+                "$ref": "cardano-babbage.json#/definitions/NativeScript"
+              }
+            }
+          },
+          "additionalProperties": false,
+          "required": ["tag", "scripts"]
+        },
+        {
+          "title": "ScriptAny",
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "any"
+              ]
+            },
+            "scripts": {
+              "type": "array",
+              "items": {
+                "$ref": "cardano-babbage.json#/definitions/NativeScript"
+              }
+            }
+          },
+          "additionalProperties": false,
+          "required": ["tag", "scripts"]
+        },
+        {
+          "title": "ScriptNOfK",
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "n_of_k"
+              ]
+            },
+            "scripts": {
+              "type": "array",
+              "items": {
+                "$ref": "cardano-babbage.json#/definitions/NativeScript"
+              }
+            },
+            "n": {
+              "$ref": "cardano-babbage.json#/definitions/UInt32"
+            }
+          },
+          "additionalProperties": false,
+          "required": [ "tag", "scripts", "n" ]
+        },
+        {
+          "title": "TimelockStart",
+          "type": "object",
+          "properties": {
+            "tag": {
+              "enum": [
+                "timelock_start"
+              ]
+            },
+            "slot": {
+              "$ref": "cardano-babbage.json#/definitions/UInt64"
```

Comment:
Even in conway cddl a slot is uint, and uint type has no upper bound limitation. Probably you need to define unit type for this case

Reply:
cddl uint is 64 bits.  
https://datatracker.ietf.org/doc/html/rfc8610#section-3.3
`"uint"  An unsigned integer (major type 0).`
https://www.rfc-editor.org/rfc/rfc8949.html#section-3.1
`Major type 0:  An unsigned integer in the range 0..2^64-1 inclusive.`

Reply:
Oh sorry totally mixed up it, sorry for that. Thanks @stevenj !